data/llama_recipe_pipeline.py

```
@@ -0,0 +1,109 @@
+        {"role": "user", "content": prompt}
+    ]
+
+    payload = {
+        "model": model,
+        "messages": messages,
+        "stream": False
+    }
+
+    for attempt in range(max_retries):
+        try:
+            response = requests.post(url, headers=headers, json=payload)
```

Ruff (0.14.2)

Probable use of `requests` call without timeout

(S113)